less/availity-spaces.less

```diff
@@ -0,0 +1,32 @@
+.spaces-tile-menu {
+  min-width: 150px;
+  max-width: 520px;
+  min-height: 180px;
+  vertical-align: top;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.spaces-tile {
+  width: 120px;
+  height: 156px;
+  position: relative;
+  float: right;
+  vertical-align: middle;
+}
+
+.spaces-tile-logo {
+  width: 100px;
+  height: 100px;
+  vertical-align: middle;
+}
+
+.spaces-tile-logo-font {
+  font-size: 14px;
+}
+
+.space-navbar-fixed-top {
```

Comment:
We need to restrict the size of images to 100x100 for consistent alignment .
UX team wants the font to be 14px for tiles label.  
space-navbar-fixed-top is needed to counter IE8 rendering issue where content of secondary nav bar is cropped at the bottom.